main.go

```diff
@@ -132,6 +132,7 @@ func main() {
 	if scorecardPublishResults == "true" {
 		err = signScorecardResult(scorecardResultsFile)
 		if err != nil {
+			fmt.Printf("Error signing scorecard result: %v", err)
```

Comment:
Thanks, could you instead please write to stderr `fmt.Fprintf(os.Stderr, "Error signing scorecard result: %v", err)`?